Source/JavaScriptCore/heap/HandleBlock.h

```diff
@@ -39,7 +39,11 @@ class HandleBlock : public DoublyLinkedListNode<HandleBlock> {
     static void destroy(HandleBlock*);
     static HandleBlock* blockFor(HandleNode*);
 
+#if defined(__mips__) || defined(__loongarch__)
+    static constexpr size_t blockSize = 16 * KB;
+#else
     static constexpr size_t blockSize = 4 * KB;
+#endif
```

Comment:
Can you explain why this is needed, please? We already have MIPS support and it doesn't seem to need this. There's also no apparent relation to anything processor-specific?